envs/monkey_zoo/blackbox/conftest.py

```diff
@@ -51,3 +57,18 @@ def pytest_runtest_setup(item):
         pytest.skip(
             "Skipping performance test because " "--run-performance-tests flag isn't specified."
         )
+
+    if item.config.getoption("--os"):
+        os = [mark.args[0] for mark in item.iter_markers(name="os")]
+        if os:
```

Comment:
Is this check necessary?

Reply:
First we check if we have `--os` and second what kind of OS is specified in the test. So I think that we need it.

Reply:
The `os` variable is a list. `if os` means "if the os list is empty". If the list is empty, the next check on line 64 will always be False, so `if os` is redundant in that case.